lib/sycamore/sycamore/tests/integration/evaluation/test_evaluate.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+from sycamore.evaluation.evaluate import QualityAssessment, Evaluate
+import sycamore
+
+
+class TestTransformEvaluate:
+    INDEX = ""
+
+    OS_CLIENT_ARGS = {
+        "hosts": [{"host": "localhost", "port": 9200}],
+        "http_compress": True,
+        "http_auth": ("admin", "admin"),
+        "use_ssl": False,
+        "verify_certs": False,
+        "ssl_assert_hostname": False,
+        "ssl_show_warn": False,
+        "timeout": 120,
+    }
+
+    OS_CONFIG = {
+        "size": 10,
+        "neural_search_k": 100,
+        "embedding_model_id": "SE1lDZABqmytCSGjsh1L",
+        "search_pipeline": "hybrid_rag_pipeline",
+        "llm": "gpt-4-turbo",
+        "context_window": "5",
+    }
+
+    @pytest.mark.skip(reason="Requires named models to configure os pipeline unless we setup the cluster on each run")
```

Comment:
Can't you simply mock those out?

Reply:
I thought it will beat the purpose of integration testing. Do you want me to add mocking here ?

Reply:
Oh, I didn't notice this was in an integration test. Is the idea that you want to skip this test in CI but still have the ability to run it manually?

Reply:
yes, that's the intension.